### PR TITLE
Add Tips for making pull request to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by vegaslon**
Add a few pointer to make it easier to get a PR through for the next guy.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,20 +22,29 @@ Contributing
   
   ```
   git remote add upstream https://github.com/kasenvr/project-athena
```
Comment (JulianGro) [inline CONTRIBUTING.md:24]:
```suggestion
  git remote add upstream https://github.com/vircadia/vircadia
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -44,4 +54,4 @@ Reporting Bugs
 Requesting a Feature
 ===
 1. Search [issues](https://github.com/kasenvr/project-athena/issues) to make sure that somebody has not already requested the same feature. 
-2. [Add](https://github.com/kasenvr/project-athena/issues/new) your request to the issues list!
+2. [Add](https://github.com/kasenvr/project-athena/issues/new) your request to the issues list!
```
Comment (JulianGro) [inline CONTRIBUTING.md:57]:
```suggestion
2. [Add](https://github.com/vircadia/vircadia/issues/new) your request to the issues list!
```

**Review by JulianGro — CHANGES_REQUESTED** [review]
I cannot give suggestions for line 10, 51 and 52 via GitHub for some reason, but one might as well go the extra mile to get those up to date.

**Review by JulianGro — APPROVED** [review]
Actually all those outdated links are changed in PR #908 , so this is fine if it gets rebased after #908 gets merged

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -21,20 +22,29 @@ Contributing
...
   *You can follow [GitHub's guide](https://help.github.com/articles/creating-a-pull-request) to find out how to create a pull request.*
   
+Tips for Pull Requests 
+===
+To make the QA process go as smoothly as possible.
+
+1. Have a basic description in your pull request. 
+2. Write a basic test plan if added features.
+3. If added any new api make sure they come with some documentation included.
+4. If accessing any external service make note of how to get a basic setup for these services or what they are based on.
```
Comment (digisomni) [inline CONTRIBUTING.md:46]:
```suggestion
1. Have a basic description in your pull request. 
2. Write a basic test plan if you are altering or adding features.
3. If a new API is added, try to make sure that some level of basic documentation on how you can utilize it is included.
4. If an added API or feature requires an external service, try to document or link to instructions on how to create a basic working setup.
```

**Review by digisomni — CHANGES_REQUESTED** [review]

**Comment by digisomni** [comment]
There are other open references PRs that will update references such as links to the repo.

**Review by daleglass — APPROVED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -20,6 +21,7 @@ Contributing
 6. Update your branch
   
   ```
+
```
Comment (digisomni) [inline CONTRIBUTING.md:24]:
```suggestion
```

Was this on purpose?

**Review by digisomni — CHANGES_REQUESTED** [review]
Just the one last thing but otherwise looks good. :)

**Review by digisomni — APPROVED** [review]

**Pull request merged**